### ISSUE TITLE
Thumbnails are not changing

### ISSUE THREAD

**Issue by SeanLeRoy** [AllenInstitute/biofile-finder#273]
The thumbnails seem to remain the same for too long after changing the file selected. I think some are duplicated too in the gallery view

**Comment by SeanLeRoy** [comment]
@BrianWhitneyAI Can you confirm this is fixed on `main` when you get your thumbnail change deployed out to production. My repro steps are:
1) Load in files with manually generated thumbnails (variance paper)
2) Change to thumbnail gallery files
3) Select an image
4) Use up and down arrow keys to change between thumbnails quickly
5) Toggle between large and small thumbnail gallery views

&

1) Load in files with auto generated thumbnails (EMT)
2) Change to thumbnail gallery files
3) Select an image
4) Use up and down arrow keys to change between thumbnails quickly
5) Toggle between large and small thumbnail gallery views

**Comment by BrianWhitneyAI** [comment]
For the generated zarr images the placeholder seems to be functioning correctly. The only issue that I was able to find with this is if you are scrolling in the gallery view and you click on 4-6 unloaded thumbnails in quick succession the thumbnail view on the side will take some time to catch up, loading one, then the next, then the next. this all occurs under half a second so not sure how relevant it is. I also noticed that the bounding box for the thumbnail view on the side for the loading box looks short. maybe a last minute change effected this/
